dapp/src/components/dashboard/containers/MainDashboard/index.jsx

```diff
@@ -124,6 +120,18 @@ class DashboardLayout extends Component {
     }
   }
 
+  checkWhereCommunityIssued = async () => {
+    const { communityAddress } = this.props
```

Comment:
this works now? I just added the `originNetwork` to the graphql

Reply:
yes it works perfect!!